source/dpp/translation/macro_.d

```diff
@@ -371,6 +371,13 @@ private auto fixCasts(R)(
             )
             return true;
 
+        if ( // macro attribute (e.g. __force) + type
+            tokens.length >= 2
+            && tokens[0].kind == Token.Kind.Identifier
```

Comment:
I could've checked that tokens[0] is present in the 'macros' array from Context. But, at this point, the '__force' Token of kind 'Identifier' has not been processed as a macro yet. This discussion is based on the unit test.